index.html

```diff
@@ -1,10 +1,167 @@
+            </p>
+        </article>
+
+        <article class="blocks-design">
+            <img src="assets/assets-for-concept/icon_web-development.svg" alt="Web Development" class="sextion-two-img">
+            <h2>Web Development </h2>
+            <p>Modern websites need to be responsive and fully functional at all sizes. We develop all of our sites with a mobile first approach and ensure that our code quality is clean, organized and performant. Already have a design but need it developed? We’re here to help!
+            </p>
+        </article>
+
+            <article class="blocks-design">
+                <img src="assets/assets-for-concept/icon_ux.svg" alt="user experience" class="sextion-two-img">
+                <h2>Web Development </h2>
+                <p>In order to build successful products you have to understand the user. We create user personas, wireframes and prototypes to test with users. This allows us to gather feedback which helps to shape the product and can quickly eliminate potential risks – saving you time and money.
+                </p>
+            </article>
+        </div>
+
+        <div class="section-two-box">
+        <article class="blocks-design">
+            <img src="assets/assets-for-concept/icon_graphic-design.svg" alt="logo-design" class="sextion-two-img">
+            <h2>Logo Design + Branding</h2>
+            <p>Nike, Apple, Coca-Cola, Starbucks. What do all of these brands have in common? Easily recognizable logos and brand identity. The best brands stand out from the crowd by having a clear and unique point of view. We will work with you to understand your vision and help create a unique brand identity and style guide for your company.</p>
+        </article>
+
+
+        <article class="blocks-design">
+            <img src="assets/assets-for-concept/icon_seo.svg" alt="SEO" class="sextion-two-img">
+            <h2>SEO</h2>
+            <p>Do you want your site to rank higher in search results? Google is constantly changing their algorithms for how they rank pages. Don’t worry about constantly playing catch-up. We will make sure your website it optimized, providing you the best opportunities for organic search exposure.</p>
+        </article>
+
+        <article class="blocks-design">
+            <img src="assets/assets-for-concept/icon_copywriting.svg" alt="" class="sextion-two-img">
+            <h2> Copy Writing </h2>
+            <p>Brand messaging needs to be clear, direct and easy to understand to be effective. A unique tone of voice will help you stand out from the crowd and build brand loyalty. We work with you to establish your brand bible and ensure your messaging is focused and targeted.</p>
+        </article>
+        </div>
+    </section>
```

Comment:
Wanneer het venster smaller wordt gemaakt gaat de tekst van de "kaartjes" erbuiten lopen. Wellicht dat de kaartjes zelf ook bij het smaller worden in twee kolommen en uiteindelijk één kolom kan komen. Dit kun je met flex netjes zelf laten oplossen